planner/optimizer/src/main/java/eu/seaclouds/platform/planner/optimizer/Optimizer.java

```diff
@@ -74,8 +71,7 @@ public Optimizer(int num, SearchMethodName name) {
 
          try {
             log.error("Calling a Replanning. The previously generated Plan will be used as a base");
-            outputPlans = optimizerReplanning.optimize(appModel,
-                  suitableCloudOffer, NUMBER_OF_PLANS_GENERATED);
+            outputPlans = optimizerReplanning.optimize(appModel, suitableCloudOffer, NUMBER_OF_PLANS_GENERATED);
             previousPlans = outputPlans;
          } catch (Error E) {
             log.error("Error optimizing the Replanning");
```

Comment:
If an exception occurs without taking any corrective action, you should bubble up the exception. If not, the error is masked. I would enclose the Error inside a OptimizerException, which extends RuntimeException.